pdns/misc.cc

```diff
@@ -796,7 +796,12 @@ int makeUNsockaddr(const std::string& path, struct sockaddr_un* ret)
   if (path.length() >= sizeof(ret->sun_path))
     return -1;
 
-  path.copy(ret->sun_path, sizeof(ret->sun_path), 0);
+  if (path.find("abns@", 0, 5) == 0) {
```

Comment:
Some comments would not hurt here. Maybe
```suggestion
  // If a Linux abstract socket is required, put its name in sun_path after a leading
  // NUL byte.
  if (path.find("abns@", 0, 5) == 0) {
```